File: Java Script - Introduction to Programming/2.1. Simple Calculations/07. 2D Rectangle Area.js
function rectangleArea([x1, y1, x2, y2]) {
  let corX = parseFloat(Math.max(x1, x2) - parseFloat(Math.min(x1, x2)))
  let corY = parseFloat(Math.max(y1, y2) - parseFloat(Math.min(y1, y2)))
  
  return `${corX * corY}\n${(corX + corY) * 2}`
}

//console.log(rectangleArea(['60', '20', '10', '50']))

File: Pyrhon - Introduction to Programming/2.1. Simple Calculations/07. 2D Rectangle Area.py
x1, y1, x2, y2 = [float(input()) for _ in range(4)]
corX = max(x1, x2) - min(x1, x2)
corY = max(y1, y2) - min(y1, y2)

print(f'{corX * corY}\n{(corX + corY) * 2}')


